README.md

```diff
@@ -126,6 +127,13 @@ docker system prune -a --volumes
 
 ### Accessing PostgreSQL Database
 
+Run in two lines:
+```bash
+docker exec -it sistema-db-1 psql -U sistema -d sistema
+SELECT * FROM public."Absence";
+```
+
+Broken into Parts
```

Comment:
What does "Broken into Parts" mean?

Reply:
Separating the commands into different lines. Rewrote for clarification